commands/files/serialfile.go

```diff
@@ -26,13 +27,17 @@ type serialFile struct {
 }
 
 func NewSerialFile(name, path string, stat os.FileInfo) (File, error) {
-	if stat.Mode()&os.ModeSymlink != 0 {
+	switch mode := stat.Mode(); {
+	case mode.IsDir() || mode.IsRegular():
+	case mode&os.ModeSymlink != 0:
 		target, err := os.Readlink(path)
 		if err != nil {
 			return nil, err
 		}
 
 		return NewLinkFile("", path, target, stat), nil
+	default:
+		return nil, fmt.Errorf("Unrecognized file format for %s: %s", stat.Name(), mode.String())
```

Comment:
would maybe do `"Unrecognized file type for"` format implies something like 'you cant add mp3's'